sous/Starter.php

```diff
@@ -22,15 +22,15 @@ public static function installTheme() {
   $drupalFinder->locateRoot(getcwd());
   $composerRoot = str_replace('-', '_', strtolower(basename($drupalFinder->getComposerRoot())));
   // Execute the Emulsify theme build based on composer create path.
-  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && php emulsify.php $composerRoot");
-  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && npm install");
+  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && php emulsify.php emulsify");
+  shell_exec ("cd web/themes/contrib/emulsify-design-system/ && yarn");
```

Comment:
We should switch this back to using npm. We were getting some inconsistency with using yarn in the shell.

Reply:
I also just noticed that we are running the npm install on the wrong theme OR we are missing this command for our custom theme... I'm not 100% sure if the base contrib `emulsify-drupal` theme needs it to run?

Either way to be safe we probably should add this line:

`shell_exec ("cd web/themes/contrib/$composerRoot/ && npm install");`

Reply:
Brian was the one that shared some knowledge with me on `yarn` being what they use on Emulsify and he said they use it over there because the yarn.lock gives you exact packages as opposed to `npm install` which pulls the latest version of packages regardless of what is in the `package.json`, we switched to using `npm ci` on TLH which does install exactly what is in `package-lock.json`, but I do think we need to standardize across our projects if you are using `yarn` or `npm ci`.  I opened a thread in the #engineers channel to discuss it.

Reply:
I'm 100% on board for standardizing as much as possible. In this case we must use `npm`. The reason being is that we were having issues with the automated build when we had this set to using `yarn`.

I'll add some thoughts to the discussion in #engineers.